MAUI/CheckBox/Getting-Started.md

```diff
@@ -0,0 +1,356 @@
+            <StackLayout>
+                  <syncfusion:SfCheckBox x:Name="checkBox"/>
+            </StackLayout>
+        </ContentPage.Content>
+    </ContentPage>
+
+{% endhighlight %}
+{% highlight c# %}
+
+    using Syncfusion.Maui.Buttons;
+
+    namespace GettingStarted
+    {
+        public partial class MainPage : ContentPage
+        {
+            public MainPage()
+            {
+                InitializeComponent();
+                StackLayout stackLayout = new StackLayout();
+                SfCheckBox checkBox = new SfCheckBox();
+                stackLayout.Children.Add(checkbox);
+                this.Content = stackLayout;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Setting caption
+
+The check box caption can be defined using the [`Text`] property of [`SfCheckBox`]. This caption normally describes the meaning of the check box and it displays next to check box.
+
+{% tabs %}
+{% highlight xaml %}
+
+    <syncfusion:SfCheckBox x:Name="checkBox" IsChecked="True" Text="CheckBox"/>
+
+{% endhighlight %}
+{% highlight c# %}
+
+    SfCheckBox checkBox = new SfCheckBox();
+    checkBox.IsChecked = true;
+    checkBox.Text = "CheckBox";
+    this.Content = checkBox;
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI CheckBox](Images/GettingStarted/checkbox.png) 
+
+## Change the check box state
+
+The three visual states of [`SfCheckBox`] are: 
+
+* Checked
+* Unchecked
+* Indeterminate
+
+![.NET MAUI CheckBox](Images/GettingStarted/tristate.png) 
+
+You can change the state of the check box using the [`IsChecked`] [`SfCheckBox`]. In checked state, a tick mark is added to the visualization of check box.
+
+<table>
+<tr>
+<td>
+<b>State</b>
+</td>
+<td>
+<b>Property</b>
+</td>
+<td>
+<b>Value</b>
+</td>
+</tr>
+<tr>
+<td>
+checked
+</td>
+<td>
+IsChecked
+</td>
+<td>
+true
+</td>
+</tr>
+<tr>
+<td>
+unchecked
+</td>
+<td>
+IsChecked
+</td>
+<td>
+false
+</td>
+</tr>
+<tr>
+<td>
+indeterminate
+</td>
+<td>
+IsChecked
+</td>
+<td>
+null
+</td>
+</tr>
+</table>
+
+N> For the check box, to report the indeterminate state, set the [`IsThreeState`] property to true.
+
+Check box can be used as a single or as a group. A single check box mostly used for a binary yes/no choice, such as "Remember me?", login scenario, or a terms of service agreement.
+
+{% tabs %}
+{% highlight xaml %}
+
+    <syncfusion:SfCheckBox x:Name="checkBox" Text="I agree to the terms of services for this site" IsChecked="True"/> 
+
+{% endhighlight %}
+{% highlight c# %}
+
+    SfCheckBox checkBox = new SfCheckBox();
+    checkBox.Text = "I agree to the terms of services for this site";
+    checkBox.IsChecked = true;
+    this.Content = checkBox;
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI CheckBox](Images/GettingStarted/termsandconditions.png) 
+
+Multiple check boxes can be used as a group for multi-select scenarios in which a user chooses one or more items from the group of choices that are not mutually exclusive.
```

Comment:
Multiple checkboxes can be used as a group for multi-select scenarios in which a user selects one or more items from the choices that are not mutually exclusive.